Cloudinary/Classes/Core/Features/Helpers/CLDExpression.swift

```diff
@@ -286,14 +291,19 @@ import Foundation
     // MARK: - provide content
     public func asString() -> String {
         
-        guard !currentKey.isEmpty && !currentValue.isEmpty else {
-            
+        guard !currentKey.isEmpty else {
+            if allSpaceAndOrDash {
```

Comment:
A specific use case of regex expression should not impact the `asString()` function
This should be handled in the Regex expression and not outside of it

Reply:
I understood that this is the new requirements (according to Patriks tests)
Meaning if the value contains "allSpaceAndOrDash" then asString() should return "_" otherwise the tests fail.